octobot_trading/exchanges/exchange_config_data.py

```diff
@@ -58,6 +58,8 @@ def __init__(self, exchange_manager):
         # number of required historical candles
         self.required_historical_candles_count = constants.DEFAULT_IGNORED_VALUE
 
+        self.future_contract_type = trading_constants.DEFAULT_SYMBOL_CONTRACT_TYPE
```

Comment:
Not sure if it's the best place to store this, the idea is to be able to set it from exchange builder

Reply:
Why not in `FutureExchange`?

Reply:
To keep storing config related values in the same place (as it's config in backtesting) however in real it's not, that's right.